lib/lightning_web/live/workflow_live/components.ex

```diff
@@ -483,4 +484,28 @@ defmodule LightningWeb.WorkflowLive.Components do
     </div>
     """
   end
+
+  slot :inner_block, required: true
+
+  def create_workflow_modal(assigns) do
```

Comment:
The create button is only enabled with a tab, mouse people will complain (:
![Screenshot from 2023-11-07 14-07-52](https://github.com/OpenFn/Lightning/assets/44991200/81c7600e-a677-4187-8983-4a8cc5559245)

Reply:
I noticed phx-change is not triggered when user types in input box.

Reply:
@benedictus-dev is this still the case?

Reply:
Yeah